### PR TITLE
[fix][doc] Changed in the IBM login url

### PR DESCRIPTION
**Pull request by 5exceptions**
### Documentation

- [X] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

Changed in the login URL of IBM cloud in doc.
image

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@5exceptions can you take a look at https://github.com/apache/pulsar-site/pull/399#issuecomment-1442688736? Thanks!

**Comment by 5exceptions** [comment]
Hi @Anonymitaet 
We are working on this and update you in next 1,2 days.
Thanks

**Comment by Anonymitaet** [comment]
@5exceptions thanks! You can submit all updates together in this PR.

**Comment by 5exceptions** [comment]
Hi @Anonymitaet

In the latest commit, we have made appropriate fixes mentioned in https://github.com/apache/pulsar-site/pull/399#issuecomment-1442688736 

(1) Fixed the display issues for version 2.11.X
(2) Fixed the display issue related to right hand side menu bar in all the versions. 

Please find the screenshot for the same below. 
V1
V2

Please review.
Thanks

**Comment by Anonymitaet** [comment]
@5exceptions thank you! Is the [1st comment](https://github.com/apache/pulsar-site/pull/399#issuecomment-1442688736) ignored?

**Comment by 5exceptions** [comment]
Sorry about it @Anonymitaet 
We have added document to master. Please review latest commit.

File: docs/deploy-ibm.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+![VM Creation Image 9](/assets/IBMCloud/VM9.png)
+
+![VM Creation Image 10](/assets/IBMCloud/VM10.png)
+
+12. Check the detail of VM in overview 
+
+![VM Creation Image 11](/assets/IBMCloud/VM11.png)
+
+13. Check the devices list, click on the menu option on the same page.
```
Comment (momo-jun) [inline docs/deploy-ibm.md:70]:
```suggestion
13. Check the devices list, and click on the menu option on the same page.
```

File: docs/deploy-ibm.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+![K8S Creation Image 8](/assets/IBMCloud/k8s8.png)
+7. Leave rest of the things as we are selecting as default. You can disable below options. Click on create and wait for provisioning of the cluster. 
+
+![K8S Creation Image 9](/assets/IBMCloud/k8s9.png)
+
+8. After the cluster is successfully provisioned, connect to the cluster. In order to connect click on the "Action" button then click on "Connect via CLI", it will give you commands, copy that and run in your VM so that we can communicate to cluster through VM. We have to configure VM for communicating to cluster in the next steps will describe how to configure VM for that.
+
+![K8S Creation Image 10](/assets/IBMCloud/k8s10.png)
+
+9. Check the created cluster list through clicking on the clusters options
```
Comment (momo-jun) [inline docs/deploy-ibm.md:111]:
```suggestion
9. Check the created cluster list by clicking on the clusters options.
```

File: docs/deploy-ibm.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+![Output of Command Helmrepo](/assets/IBMCloud/Helmrepo.png)
+
+2. Clone the Pulsar Helm chart repository, go inside the pulsar-helm-chart directory
+
+```bash
+git clone https://github.com/apache/pulsar-helm-chart   
+cd pulsar-helm-chart
+```
+
+3. Run the script`prepare_helm_release.sh`to create secrets required for installing the Apache Pulsar Helm chart. The username`pulsar`and password`pulsar`are used for logging into the Grafana dashboard and Pulsar Manager.
```
Comment (momo-jun) [inline docs/deploy-ibm.md:191]:
```suggestion
3. Run the script`prepare_helm_release.sh`to create secrets required for installing the Apache Pulsar Helm chart. The username`pulsar`and password `pulsar` are used for logging into the Grafana dashboard and Pulsar Manager.
```

File: docs/deploy-ibm.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+Output
+![Output of Command HelmInstall](/assets/IBMCloud/HelmInstall.png)
+
+5. Check the status of all pods.
+
+```bash 
+kubectl get pods
+```
+If all pods start up successfully, you can see that `STATUS`is changed to`Running`or`Completed`.
```
Comment (momo-jun) [inline docs/deploy-ibm.md:226]:
```suggestion
If all pods start up successfully, you can see that `STATUS` is changed to `Running` or `Completed`.
```

File: docs/deploy-ibm.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+
+Output
+
+![Output of Command Services](/assets/IBMCloud/Services.png)
+
+The output shows both `services URL` and `broker URL`. 
+Proxy external IPs are the ports changed just now:
+- Service URL port is 80.
+- Broker URL port is 6650. This is default.
+If you execute all the commands successfully, you can use the Pulsar client to connect to ?? clusters and produce and consume messages through proxy external IPs.
```
Comment (momo-jun) [inline docs/deploy-ibm.md:246]: Is this an unfinished paragraph with "?? clusters"?

Comment (Anonymitaet) [reply docs/deploy-ibm.md:246]: ping @5exceptions

Comment (5exceptions) [reply docs/deploy-ibm.md]: Thanks @momo-jun , @Anonymitaet 
We have fixed all the reported issue regarding the formatting. Please review the latest commit.

**Review by Anonymitaet — APPROVED** [review]

**Review by Anonymitaet — COMMENTED** [review]
Add this file ID to https://github.com/apache/pulsar-site/blob/main/sidebars.json?

**Comment by 5exceptions** [comment]
@Anonymitaet 
Added this file ID to sidebars.json. Please review latest commit.
Thanks

**Review by momo-jun — APPROVED** [review]
LGTM. @5exceptions Thanks for adding the docs! 
We can continuously improve it in terms of style and wording.

**Comment by Anonymitaet** [comment]
ping @5exceptions to take a look at https://github.com/apache/pulsar-site/pull/440#pullrequestreview-1325823955

**Comment by Anonymitaet** [comment]
@5exceptions please resolve conflicts

**Comment by 5exceptions** [comment]
@Anonymitaet : Conflicts are resolved. Please review.

**Pull request merged**